core/domain/story_domain.py

```diff
@@ -0,0 +1,369 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Domain objects relating to stories."""
+
+from constants import constants
+from core.platform import models
+import feconf
+
+(story_models,) = models.Registry.import_models([models.NAMES.story])
```

Comment:
Why import this if you don't use it?

@apb7 why isn't this flagged by the linter?

Reply:
Done.